wcfsetup/install/files/lib/system/comment/manager/AbstractCommentManager.class.php

```diff
@@ -153,10 +154,14 @@ public function canDeleteResponse(CommentResponse $response)
     }
 
     /**
+     * @param UserProfile|null $user
```

Comment:
Do not add PhpDoc that is the same as the parameter type, that’s simply redundant and at risk of getting out of sync. Also, the parameter should be typed in the interface so it makes even less sense.